ddtrace/profile/__main__.py

```diff
@@ -0,0 +1,20 @@
+"""Main command pyddprofile."""
+import os
+import sys
+
+
+def main():
+    bootstrap_dir = os.path.join(os.path.dirname(__file__), "bootstrap")
```

Comment:
might not hurt to do `os.path.abspath()` here, help get around any potential issues